### ISSUE TITLE
map is getting worse

### ISSUE THREAD

**Issue by eeric** [HikariTJU/LD#8]
I caculated by below formula：
loss = loss_qfl + loss_bbox + loss_dfl + loss_ld
the result was that map was worse.
why？

**Comment by HikariTJU** [comment]
No details, hard to judge.

**Comment by eeric** [comment]
oh, sorry，15 epoch ahead, map was lower, now util trainning end, map was lower than feature distillation, while ap50 was higher.
you yet used above formula, did you?
or the 4 items weights in the formula  how to set
besides, loss_ld was always large.

[root][05-10 01:06:28]INFO:train|Epoch257/300|Iter395049(40/1543)| lr:1.00e-04| loss_qfl:0.0892| loss_bbox:0.3611| loss_ld:1.3329| loss_dfl:0.3016| 
[root][05-10 01:06:32]INFO:train|Epoch257/300|Iter395059(50/1543)| lr:1.00e-04| loss_qfl:0.0898| loss_bbox:0.3447| loss_ld:1.0232| loss_dfl:0.3002|

**Comment by HikariTJU** [comment]
0.25 * loss_ld + 0.25 * loss_dfl + 1.0 * loss_qfl + 2.0 * loss_bbox

**Comment by eeric** [comment]
good advice

**Comment by eeric** [comment]
> 0.25 * loss_ld + 0.25 * loss_dfl + 1.0 * loss_qfl + 2.0 * loss_bbox

@HikariTJU 
I tried by two set of 4 loss weight coefficient, map was lower than feature based loss distillation, unfortunately.

**Comment by Zzh-tju** [comment]
@eeric 
More experiments and analysis about LD and feature imitation is coming soon.

**Comment by twmht** [comment]
@HikariTJU 

How you tune these parameters?
